<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Greenhouse Monitor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/myimage/helios.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fan </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/myimage/fan_off.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/myimage/heater_off.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFan_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heater</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Water</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/myimage/water_off.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gigi</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NDA</string>
    </property>
   </widget>
   <widget class="QWidget" name="rightPanel" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>684</width>
      <height>781</height>
     </rect>
    </property>
    <widget class="QWidget" name="chartContainer" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>664</width>
       <height>520</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupAI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>664</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>AI Prediction &amp; Status</string>
     </property>
     <widget class="QLabel" name="label_tempNow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Temp now: -- °C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_humNow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Humidity: -- %</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_uvNow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>UV Index: --</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tempPredTitle">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Predicted temp:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tempPred">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-- °C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_aiNote">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>280</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>(Auto mode uses AI + rules)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_modeHint">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>300</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Press M to toggle Manual/AUTO</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
